tests/test_mocks.py

```
@@ -0,0 +1,117 @@
+
+"""
+    input_file.write_text(test_content)
+    
+    with patch('translate_srt.client.chat.completions.create') as mock_create:
+        # Mock OpenAI response
+        mock_response = Mock()
+        mock_response.choices = [Mock(message=Mock(content="Translated subtitle"))]
+        mock_create.return_value = mock_response
+            
+        # Mock translate_text to return a string instead of coroutine
+        async def mock_translate_text(*args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for private function `mock_translate_text`

Add return type annotation: `str`

(ANN202)